src/test/add.test.ts

```diff
@@ -0,0 +1,48 @@
+import add from '../add';
+import chai, { should, expect } from 'chai';
+
+chai.should();
+
+describe('add function', () => {
+  it('Sum should be equal to 3', () => {
+    add(1, 2).should.equal(3);
+  });
+});
+
+describe('Null and Zero check', () => {
```

Comment:
in general it is good to have tests in __tests__ folder

Reply:
Okay Ram.